src/components/Modals/AddToPlaylistModal/AddToPlaylistModal.tsx

```diff
@@ -48,11 +48,11 @@ export const AddToPlaylistModal = (props: Props) => {
   const _handlePlaylistPress = async (playlist: Playlist) => {
     const { item } = addToPlaylist
     OmniAural.pageIsLoadingShow()
-    if (item.clipId) {
```

Comment:
Typescript struggles with these types of checks when you have union types, it requires you to use `in`, discriminating unions, or custom type guards

Reply:
Ok, I've been making this mistake for a while. I'll start using `in` for these kinds of type checks.